airflow/providers/amazon/aws/waiters/dynamodb.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": 2,
+    "waiters": {
+        "export_table": {
+            "operation": "DescribeExport",
+            "delay": 30,
+            "maxAttempts": 60,
+            "acceptors": [
+                {
+                    "matcher": "path",
+                    "expected": "COMPLETED",
+                    "argument": "ExportDescription.ExportStatus",
```

Comment:
According to my notes, I think the current issue is because these "argument" fields need to be all lowercase.  Documentation on the BOTO side for these is pretty hard to find, I might be mistaken, but if you are still running into that ` 'dynamodb.ServiceResource' object has no attribute 'describe_export'` issue. 

 If that is the case, can you please also update the waiters README to make that more obvious??

Reply:
@ferruzzi I'm still facing the same issue post changes to Arguments. Updated json config looks like:
```
{
    "version": 2,
    "waiters": {
        "export_table": {
            "operation": "DescribeExport",
            "delay": 30,
            "maxAttempts": 60,
            "acceptors": [
                {
                    "matcher": "path",
                    "expected": "COMPLETED",
                    "argument": "exportdescription.exportstatus",
                    "state": "success"
                },
                {
                    "matcher": "path",
                    "expected": "FAILED",
                    "argument": "exportdescription.exportstatus",
                    "state": "failure"
                },
                {
                    "matcher": "path",
                    "expected": "IN_PROGRESS",
                    "argument": "exportdescription.exportstatus",
                    "state": "retry"
                }
            ]
        }
    }
}
```
Would it make sense to change the logic to polling on the `DescribeExport` API, until there is a fix for the waiters in botocore?

Reply:
I was mistaken on the lowercase suggestion.  Looking through a handful of other examples of official waiters in the botocore package, (see [EC2](https://github.com/boto/botocore/blob/develop/botocore/data/ec2/2016-11-15/waiters-2.json#L81) for example)  they all seem to match the case exactly as the [boto API docs](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/dynamodb/client/describe_export.html) have it,, so in this instance PascalCase would be the right answer.  Sorry about that.

I have one last idea if you are willing to try it out, but I won't block the PR longer for this.

I think the root issue is that the DynamoDB hook is creating a `resource_type` instead of a `client_type` and, as the erro message says, the ServiceResource doesn't have a method called `describe_export`.  But the BaseAwsHook `get_waiter` method accepts a client as a parameter, so we could try that.  Something like:

```
waiter = self.hook.get_waiter(waiter_name=CUSTOM_WAITER_NAME, client=boto3.client('dynamodb'))
waiter.wait(ExportArn=response.get("ExportArn"))
```


If that doesn't work, then I'm willing to concede this one and we can do a polling loop here until I et time to look deeper into why this one won't cooperate.

Reply:
@ferruzzi This worked! Thanks :)